src/domUpdates.js

```diff
@@ -0,0 +1,36 @@
+import dayjs from 'dayjs'
+
+//querySelectors
+const tripList = document.querySelector('.trip-list')
+const greeting = document.querySelector('.greeting')
+const annualSpending = document.querySelector('.annual-expenditure')
+
+const domUpdates = {
+    userGreeting(user) {
+        let theUser = user.getFirstName()
+        greeting.innerText = `Welcome back, ${theUser}!`
+    },
+
+    displayAnnualSpending(user) {
+        annualSpending.innerText = `You've spent ${user.annualSpent()} in ${dayjs().format('YYYY')}`
```

Comment:
Same question. Does this need the whole user object if it only uses the annual spent value?